react-carousel-component/my-app/src/Carousel.tsx

```diff
@@ -0,0 +1,112 @@
+import {
+  AiOutlineRight,
+  AiOutlineLeft,
+  AiOutlinePayCircle,
+} from 'react-icons/ai';
+import { useEffect, useState } from 'react';
+
+type Prop = {
+  images: {
+    imgLocation: string;
+    id: number;
+  }[];
+};
+//added note to save and commit
+export default function Carousel({ images }: Prop) {
+  const [activeIndex, setActiveIndex] = useState(0);
+
+  useEffect(() => {
+    let interval = setInterval(() => {
+      setActiveIndex((activeIndex + 1) % images.length);
+    }, 3000);
+    return () => clearInterval(interval);
+  }, [activeIndex, images.length]);
+
+  return (
+    <div className="container">
+      <div className="main row">
+        <ArrowButton
+          right={false}
```

Comment:
boolean props have interesting syntax. If you leave off the prop entirely, its value will be false. If you specify it without any value its value will be true. So you can do this:
`<ArrowButton />` displays with `right = falsy`
`<ArrowButton right />` displays with `right = true`